### DISCUSSION TITLE
Status on Power (relay) after Powercycle needed on Rules

### DISCUSSION THREAD

**Discussion by tsunfinger** [arendst/Tasmota#22654] [General]
I have a special setup on rules. The challenge is the unknown status on Power (relay) after a powercycle. I store some information on mem, but it only works on restart - not on powercycle.
Is there any option on rules to get out the current status on power (without switching)?

**Comment by sfromis** [comment]
Try triggers like `Power1#Boot`
https://tasmota.github.io/docs/Rules/#examples-of-available-triggers

**Reply by tsunfinger** [reply]
Thanks for hint, it's working.
